chart/templates/mayastor/io/io-engine-daemonset.yaml

```diff
@@ -64,10 +64,14 @@ spec:
         env:
         - name: RUST_LOG
           value: {{ .Values.io_engine.logLevel }}
-        - name: NVME_QPAIR_CONNECT_ASYNC
-          value: "true"
```

Comment:
Is this change intentional?

Reply:
Yes, it's always enabled for sometime now